### PR TITLE
feat: add create-react-app completion spec

### PR DESCRIPTION
**Pull request by symsmith**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature: Add `create-react-app` completion spec

### CODE REVIEW

File: src/create-react-app.ts [hunk 1]
```diff
@@ -0,0 +1,104 @@
...
+  },
+  options: [
+    // template
+    {
+      name: "--template",
+      description:
+        'Specify a template for the created project (a custom template on npm (search for "cra-template-*"), a relative path, or an archive (.tgz or .tar.gz))',
+      args: {
+        name: "name or url",
+        isOptional: false,
```
Comment (QuiiBz) [inline src/create-react-app.ts:52]: You can remove `isOptional` when it's `false` - it's the default value!

**Review by QuiiBz — CHANGES_REQUESTED** [review]
Thanks a lot for this contribution! Only one small change and we'll be ready to merge!

**Comment by QuiiBz** [comment]
And there's also the lint script which is failing - could you run the `lint:fix` script?

**Comment by symsmith** [comment]
It looks like the linter is failing on `dangerfile-greeting.ts`, which I haven't edited.

Log:
```
/home/runner/work/autocomplete/autocomplete/dangerfile-greeting.ts
  26:69  error  Parsing error: ',' expected
```

**Comment by symsmith** [comment]
The problem comes from unescaped backticks, I can commit the fix if you want.

**Comment by QuiiBz** [comment]
> The problem comes from unescaped backticks, I can commit the fix if you want.

I pushed a fix yesterday to `master`, so you can simply rebase/merge!

**Comment by symsmith** [comment]
Thanks, should be all good now!

**Review by QuiiBz — APPROVED** [review]
LGTM, thank you so much for this contribution!

**Pull request merged**